packages/diff-so-fancy/build.sh

```diff
@@ -0,0 +1,24 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/so-fancy/diff-so-fancy"
+TERMUX_PKG_DESCRIPTION="Good-lookin' diffs. Actually… nah… The best-lookin' diffs"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_VERSION="1.4.4"
+TERMUX_PKG_SRCURL="${TERMUX_PKG_HOMEPAGE}/archive/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="3eac2cfb3b1de9d14b6a712941985d6b240b7f3726c94a5e337317c7161e869d"
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS=perl
+TERMUX_PKG_RECOMMENDS=git
```

Comment:
```suggestion
TERMUX_PKG_RECOMMENDS="git"
```
`TERMUX_PKG_RECOMMENDS` and `DEPENDS` are list type control variables and should as such be quoted in case another entry is added to them down the line.
I know we haven't always enforced that in the past, I've noted it for the next revision of the wiki and package build linter.

Reply:
Changed in 3f36bc80c